plugins/outputs/influxdb/README.md

```diff
@@ -43,6 +43,9 @@ This plugin writes to [InfluxDB](https://www.influxdb.com) via HTTP or UDP.
 
   ## HTTP Proxy Config
   # http_proxy = "http://corporate.proxy:3128"
+
+  ## Compress each HTTP request payload using GZIP, defaults to false.
+  # gzip = false
```

Comment:
Let's change the option to be `content_encoding = "gzip"` supporting only `gzip`, this will allow us to potentially add more encodings later and IMO reads better.